src/pages/dashboard/dashboard.scss

```diff
@@ -251,5 +251,5 @@
 
 :focus {
   outline: 2px dotted;
-  color: $color-slate-9;
+  outline-color: $color-slate-9;
 }
```

Comment:
I'm unsure about this block more generally. Without a class or any other scoping, this will affect *all* elements on the page. Is that what we want? If so, it probably belongs in a global stylesheet, not in this partial `dashboard.scss` file.

Reply:
this is a good point. The effects of this have bled to the rest of the site:
![image](https://user-images.githubusercontent.com/16123225/79289904-7a898280-7e7f-11ea-8c85-5d48c3f30afa.png)